NPM install @nangohq/frontend is broken

Installing `@nanghoq/frontend` is broken. To reproduce:
```
robin@MacBook-Air ~ % mkdir test
robin@MacBook-Air ~ % npm i @nangohq/frontend
npm ERR! code 127
npm ERR! path /Users/robin/node_modules/@nangohq/frontend
npm ERR! command failed
npm ERR! command sh -c npm run build
npm ERR! > @nangohq/frontend@0.15.0 build
npm ERR! > tsc --sourceMap
npm ERR! sh: tsc: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/robin/.npm/_logs/2023-05-01T20_25_24_563Z-debug-0.log
```

--- a/packages/frontend/package.json
+++ b/packages/frontend/package.json
@@ -1,6 +1,6 @@
 {
     "name": "@nangohq/frontend",
-    "version": "0.15.0",
+    "version": "0.15.1",
     "description": "Nango's frontend library for OAuth handling.",
     "type": "module",
     "main": "dist/index.js",
@@ -11,10 +11,6 @@
         "type": "git",
         "url": "https://github.com/NangoHQ/nango",
         "directory": "packages/frontend"
-    },
-    "scripts": {
-        "build": "tsc --sourceMap",
-        "postinstall": "npm run build"
     },
     "license": "SEE LICENSE IN LICENSE FILE IN GIT REPOSITORY",
     "dependencies": {},

